### PR TITLE
Create project-gallery.html

### PR DESCRIPTION
**Pull request by GoldinGuy**
Add project gallery component for Hacktoberfest!

## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [X] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Added new project gallery component

## Related Tickets & Documents

N/A

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image

## Added to documentation?

- [ ] 📜 readme
- [X] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

[link](https://tenor.com/view/hollow-knight-hollow-dance-the-hollow-gif-17512843)

### CODE REVIEW

File: components/project-gallery.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
+  <section class="bg-teal-50" id="projects">
```
Comment (bobbyiliev) [inline components/project-gallery.html]: Looks good! There are just some extra whitespace throughout the file, would you mind formatting the document and removing them?

File: components/project-gallery.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+         <div
+            class="px-3 pt-12 pb-0 mx-auto sm:px-6 sm:pt-10 md:max-w-4xl md:pt-10 lg:max-w-6xl overflow-hidden"
+            >
```
Comment (bobbyiliev) [inline components/project-gallery.html]: One more small remark, can you put the classes on the same line as the tags? Thanks 🙌

**Review by bobbyiliev — CHANGES_REQUESTED** [review]

File: components/project-gallery.html (hunk 1 above)
Comment (GoldinGuy) [reply components/project-gallery.html]: Will do

File: components/project-gallery.html (hunk 2 above)
Comment (GoldinGuy) [reply components/project-gallery.html]: Sure thing!

**Comment by GoldinGuy** [comment]
Requested changes have been made. Let me know if anything else needs work

**Review by bobbyiliev — APPROVED** [review]

**Pull request merged**